apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -1726,5 +1726,8 @@
         "theme": {
             "name": "{{ console.theme }}"
         }
-    }
+    }{{ "," if console.validate_id_token_issuer is defined }}
```

Comment:
Let's add this under `idpConfigs`

Reply:
Since this new config enables/ disbales id token issuer validation, it does not belong to `idpConfigs`.

Reply:
Also, will it be okay to add this config outside the entire config object?

cc: @brionmario

Reply:
I'm guessing this is a SDK related config? @dhaura 
If so, as @pavinduLakshan has suggested, that should go in `idpConfigs`

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/5050/commits/4e0f4328aad4466579342d39c0d658788c7a6b1b